src/utils/trust/mapper.to.page.ts

```diff
@@ -19,6 +19,20 @@ const mapDetailToPage = (
   };
 };
 
+
+const mapTrustWhoIsInvolvedToPage = (
```

Comment:
I removed this file
could you please create `who.Is.Involved.mapper.ts` and put content from here
please align test file

Reply:
Reverted the changes in this file as it turned out they were unneeded